modules/installation-guide/examples/system-variables.adoc

```diff
@@ -80,12 +80,12 @@ pass:[<!-- vale off -->]
  Environment Variable Name,Default value, Description 
  
  `+SCHEDULE_CORE__POOL__SIZE+`,"`+10+`","{prod-short} extensions can be scheduled executions on a time basis. This configures the size of the thread pool allocated to extensions that are launched on a recurring schedule." 
- `+DB_SCHEMA_FLYWAY_BASELINE_ENABLED+`,"`+true+`","DB initialization and migration configuration" 
- `+DB_SCHEMA_FLYWAY_BASELINE_VERSION+`,"`+5.0.0.8.1+`","DBinitialization and migration configuration" 
- `+DB_SCHEMA_FLYWAY_SCRIPTS_PREFIX+`,"","DBinitializationand migration configuration" 
- `+DB_SCHEMA_FLYWAY_SCRIPTS_SUFFIX+`,"`+.sql+`","DBinitializationandmigration configuration" 
- `+DB_SCHEMA_FLYWAY_SCRIPTS_VERSION__SEPARATOR+`,"`+__+`","DBinitializationandmigrationconfiguration" 
- `+DB_SCHEMA_FLYWAY_SCRIPTS_LOCATIONS+`,"`+classpath:che-schema+`","DBinitializationandmigrationconfiguration" 
+ `+DB_SCHEMA_FLYWAY_BASELINE_ENABLED+`,"`+true+`","DB initialization and migration configuration If true, ignore scripts up to the version configured by baseline.version." 
```

Comment:
`If true, ignore scripts up to and including the value of DB_SCHEMA_FLYWAY_BASELINE_VERSION`